src/js/globals.js

```diff
@@ -27,14 +27,26 @@
 
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis
 
-const globals = (( ) => {
+const realGlobals = (( ) => {
     // jshint ignore:start
     if ( typeof globalThis !== 'undefined' ) { return globalThis; }
     if ( typeof self !== 'undefined' ) { return self; }
     if ( typeof global !== 'undefined' ) { return global; }
     // jshint ignore:end
+    throw new Error('unable to locate global object');
 })();
 
+// Make a shallow copy so as not to pollute the global namespace.
+const globals = Object.assign({}, realGlobals);
+
+if ( 'WebAssembly' in realGlobals ) {
```

Comment:
The `WebAssembly` and `URLSearchParams` properties are copied explicitly since they are not [keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) of the effective `globalThis`.

Reply:
By the way, `WebAssembly` and `URLSearchParams` could totally be used directly without `globals.`. They are both available in both browser and Node.js environments, and when they are not the code needs to deal with it anyway. I'm guessing the way it's done at the moment is to satisfy ESLint.